app/containers/MessageActions/index.tsx

```diff
@@ -343,15 +344,18 @@ const MessageActions = React.memo(
 				}
 			};
 
-			const handleReport = async (message: TAnyMessageModel) => {
+			const handleReport = (message: TAnyMessageModel) => {
 				logEvent(events.ROOM_MSG_ACTION_REPORT);
-				try {
-					await Services.reportMessage(message.id);
-					Alert.alert(I18n.t('Message_Reported'));
-				} catch (e) {
-					logEvent(events.ROOM_MSG_ACTION_REPORT_F);
-					log(e);
-				}
+				let description = message.msg ?? '';
+				description = description.length <= 20 ? description : `${description.slice(0, 20)} ...`;
```

Comment:
This value of 20, where did it come from?

Reply:
Sir this is an arbitary value I used to trim messages, Should I display the whole message?

Reply:
Yes, we cannot take an arbitrary value as true, this would create several loopholes.
Either it shows everything, or it doesn't show everything, while still giving the option to see everything.

Because for you 20 characters may be enough, but for someone else it should be 30 and so on.